From 4a3dceb097fceefc13c4471707cd40944dd07ffb Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Fri, 17 Aug 2018 23:27:54 +0000
Subject: [PATCH] OS-7033 Autogenerate Intel pcbe values from perfmon data (fix
 custr) Reviewed by: Jason King <jbk@joyent.com> Approved by: Jason King
 <jbk@joyent.com>

---
 usr/src/tools/cpcgen/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/tools/cpcgen/Makefile b/usr/src/tools/cpcgen/Makefile
index dc50c0b1e0..23f412439a 100644
--- a/usr/src/tools/cpcgen/Makefile
+++ b/usr/src/tools/cpcgen/Makefile
@@ -35,7 +35,7 @@ $(PROG): $(OBJS)
 %.o: $(SRC)/lib/json_nvlist/%.c
 	$(COMPILE.c) $<
 
-%.o: $(SRC)/lib/libcmdutils/common/%.c
+%.o: $(SRC)/lib/libcustr/common/%.c
 	$(COMPILE.c) $<
 
 clean:
-- 
2.21.0

